packages/editor-ui/src/components/ContactPromptModal.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+	<Modal
+		:name="modalName"
+		:eventBus="modalBus"
+		:center="true"
+		:closeOnPressEscape="false"
+		:customClass="'contact-prompt-modal'"
+		:beforeClose="closeDialog"
+		title="You're a power user"
+		minWidth="460px"
+		maxWidth="460px"
+	>
+		<template slot="header">
+			<h2 :class="$style.title">You’re a power user 💪</h2>
+		</template>
+		<template v-slot:content>
+			<div :class="$style.description">Your experience with n8n can help us improve - for you and our entire community.</div>
```

Comment:
can you try to use the text component https://n8n-io.github.io/n8n-storybook/?path=/story/atoms-text--text

Reply:
Done after your directions from yesterday :)